### PR TITLE
add explainer for canLoadOpaqueURL() API

### PR DESCRIPTION
**Pull request by blu25**
This adds an explainer outlining a new API for fenced frames to tell if an opaque ads fenced frame will be allowed to load in a given context.

This change has gone through CMA pre-review and can be merged in.

### CODE REVIEW

**Comment by domfarolino** [comment]
Is this PR on Eric's radar, so once we get approval we can land it?

**Comment by shivanigithub** [comment]
Yes I believe this is approved. I am about to add a few comments though.

File: explainer/README.md [hunk 1]
```diff
@@ -131,6 +131,29 @@ There are many channels between the fenced frame tree and the other frames that
...
+
+All fenced frame related functions will live in its own class, in the same way that iframe-related funcionality lives in HTMLIFrameElement.
+
+#### Opaque-Ads Can Load API
+
+There are various reasons a fenced frame embedded as an opaque ad could refuse to load in a page. For example, if the page is not in a secure context, or if CSPEE is specified in the embedding frame, the fenced frame will refuse to load. This is a lot for a developer to keep track of.
+
+If the process of getting an ad in the page is complex or expensive, there needs to be a way to ensure that the resulting ad will actually end up in the page before the expensive process begins.
+
+An static API method will be introuced to the HTMLFencedFrameElement class to check this. It will be called before a fenced frame is created, and will return a boolean, true if an opaque-ads fenced frame can load, and false if not.
```
Comment (shivanigithub) [inline explainer/README.md:144]: Can we rephrase this to something like below for more clarity:
"No fenced frame will be created on calling this API and it can be invoked before actually attempting to create a fenced frame"

Comment (shivanigithub) [inline explainer/README.md:144]: s/An/A

File: explainer/README.md [hunk 2]
```diff
@@ -131,6 +131,29 @@ There are many channels between the fenced frame tree and the other frames that
...
+##### Example usage
+
+```
+HTMLFencedFrameElement.canLoadOpaqueURL();
+```
+```
+> true
+```
+
+This is called synchronously, and will look at which ever frame its execution context sits in.
```
Comment (shivanigithub) [inline explainer/README.md:155]: "... and will look at the execution context of the frame invoking the API"?
